core/modules/config/config.admin.inc

```diff
@@ -230,6 +230,9 @@ function config_import_full_form(array $form, array &$form_state) {
   $form['submit'] = array(
     '#type' => 'submit',
     '#value' => t('Stage import'),
+    '#attributes' => array(
+      'class' => array('button-primary'),
+    ),
```

Comment:
We should wrap these submit buttons in a `$form['actions']['#type'] = 'actions';` wrapper, that will add the button-primary class automatically to the first button.